File: BankingManagementSystem.Angular/src/app/user-management/new-user/newUserDto.ts
import { ClaimDto } from "../claims/claimDto";
import { RoleDto } from "../roles/roleDto";
import { UserDto } from "../user-list/userDto";

export class NewUserDto{
  public userName:string = "admin@admin.com";
  public email:string = "admin@admin.com"
  public password:string = "1234";
  public confirmPassword: string = "1234";
  public roles: RoleDto[] | [] = [];
  public claims: ClaimDto[] | [] = [];
}

File: BankingManagementSystem/Dto/NewUserDto.cs
using System.Collections.Generic;
using System.Diagnostics.CodeAnalysis;

namespace BankingManagementSystem.Dto
{
	public class NewUserDto
	{
		[NotNull]public string Username { get; set; }

		[NotNull]public string Email { get; set; }

		[NotNull]public string Password { get; set; }
		
		public List<RoleDto> Roles { get; set; }

		public List<ClaimDto> Claims { get; set; }
	}
}

